tests/drive_failure/drive_failure_test.go

```diff
@@ -0,0 +1,102 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/node"
+	. "github.com/portworx/torpedo/tests"
+)
+
+func TestDriveFailure(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Torpedo: DriveFailure")
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+var _ = Describe("Induce drive failure on of the nodes", func() {
+	testName := "drivefailure"
+	It("has to schedule apps and induce a drive failure on one of the nodes", func() {
+		var err error
+		contexts := ScheduleAndValidate(testName)
+
+		Step("get nodes for all apps in test and induce drive failure on one of the nodes", func() {
+			for _, ctx := range contexts {
+				var (
+					drives               []string
+					appNodes             []node.Node
+					nodeWithDrive        node.Node
+					driveToFail, driveID string
+				)
+
+				Step(fmt.Sprintf("get nodes where %s app is running", ctx.App.Key), func() {
+					appNodes, err = Inst().S.GetNodesForApp(ctx)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(appNodes).NotTo(BeEmpty())
+					nodeWithDrive = appNodes[0]
```

Comment:
Instead of picking the first one, we can do this on a quorum on nodes.

Reply:
I don't see a reason why we need to fail drives on more than one node. 
What I can check for is if the node whose drive I am failing has the replica of the volume being used.

Reply:
Yeah. We could end up off-lining drives on a node which doesn't have the replica of the volume.

Reply:
Currently it assumes that the volume replica is 3 and the cluster size is 3 (default settings).
I have created an issue to add a new API to handle the generic case where replica may not exist on all nodes - #83